runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -79,29 +82,47 @@ public void handleFailure(IReconfigurationHandlerPolicy failureHandlerPolicy,
 
     /**
      * Takes in the existing layout and a set of failed nodes.
-     * It first generates a new layout by removing the failed nodes from the existing layout.
-     * It then seals the epoch to prevent any client from accessing the stale layout.
-     * Finally we run paxos to update all servers with the new layout.
+     * Dispatches a workflow request to heal the node.
      *
      * @param healingHandlerPolicy Policy dictating healing of the reviving node.
      * @param currentLayout        The current layout
-     * @param healedServers        Set of healed server addresses
+     * @param healedServer         Healed server address
      * @throws QuorumUnreachableException  if reconfiguration not accepted by quorum.
-     * @throws OutrankedException          if reconfiguration consensus attempt outranked.
      * @throws LayoutModificationException if attempting to reconfigure with invalid layout.
      */
     public void handleHealing(IReconfigurationHandlerPolicy healingHandlerPolicy,
                               Layout currentLayout,
-                              Set<String> healedServers)
-            throws QuorumUnreachableException, OutrankedException, LayoutModificationException {
+                              String healedServer) {
```

Comment:
It's not clear that this method belongs in LayoutManagementView... healNode does, but not handleHealing

Reply:
Right, this does not belong here. I moved this method to ReconfigurationEventHandler.